plugins/inventory/now.py

```diff
@@ -423,7 +423,7 @@ def _get_instance_from_env(self):
             username=os.getenv("SN_USERNAME"),
             password=os.getenv("SN_PASSWORD"),
             client_id=os.getenv("SN_CLIENT_ID"),
-            client_secret=os.getenv("SN_SECRET_ID"),
```

Comment:
For backward compatibility, maybe we should use next code:
`client_secret=os.getenv("SN_CLIENT_SECRET") or os.getenv("SN_SECRET_ID")`

Since there should be some users, that already used old implementation.